content/docs/0.9.x/reference/bridge/manage_sequences/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Manage Sequences
+description: Manage sequences from the Bridge
+weight: 40
+keywords: [0.9.x-bridge]
+---
+
+The sequence view shows all executed sequences for the selected project.
+
+The list on the left side gives a brief overview of all sequences and can be filtered. By clicking one of the sequences, the detail for it opens.
+This detailed view shows every executed task for the selected sequence in a specific stage. It includes information like the related events, the result, and the payload.
+
+For evaluation sequences, the Bridge will also provide a link to the evaluation board.<br/>
+For delivery sequences, the Bridge will provide a link to the deployed artifact.
+
+The default selection when opening the detail is the last executed stage. Select a different one by clicking on the stage name in the timeline.
+
+{{< popup_image
+link="./assets/running-sequence.png"
+caption="Sequences">}}
+
+## Filtering
+
+Filtering provides a way of getting a better overview of the sequences. Filter by using the search bar or by selecting the items from the filter list.
+Filters can apply to:
+* Service - all services created for the project
```

Comment:
```suggestion

* Service - all services created for the project
```
Could you add a line break before the list? Else it will look like this.
![image](https://user-images.githubusercontent.com/11599148/137336650-8b607d9d-3240-4ac6-baf6-e5169826e223.png)